css/media.css

```diff
@@ -0,0 +1,413 @@
+@media screen and (max-width: 1100px) {
+  .container {
+    max-width: 970px;
+  }
+}
+
+@media screen and (max-width: 1024px) {
+  .container {
+    max-width: 930px;
+  }
+
+  .header {
+    padding: 27px 0 8px 0;
+  }
+
+  .header__logo {
+    margin-right: 30px;
+    margin-top: 5px;
+  }
+
+  .header__logo-img {
+    width: 61px;
+    height: 55px;
+  }
+
+  .navbar__right-side {
+    margin-top: 3px;
+  }
+
+  .navbar__right-side .navbar__item:first-child {
+    margin-right: 32px;
+  }
+
+  .navbar__right-side .navbar__item:nth-child(2) {
+    margin-right: 28px;
+  }
+
+  .navbar__right-side .navbar__item:nth-child(3) {
+    margin-right: 24px;
+  }
+
+  .navbar__right-side .navbar__item:last-child {
+    margin-right: 0;
+  }
+
+  .search-form {
+    width: 158px;
+    margin-top: 6px; /* for PP */
+  }
+  
+  .search-form__input {
+    width: 140px;
+  }
+
+  /* ------------------------- */
+
+  .market-section {
+    padding-top: 32px;
+  }
+  
+  .banner {
+    padding: 51px 0 0 53px ;
+    min-height: 253px;
+    background-size: cover;
+  }
+  
+  .banner__title {
+    max-width: 210px;
+    margin-bottom: 9px;
+  }
+  
+  .banner__link-more::after { /* Разобраться почему не управляется ширина */
+    margin-left: 15px;
+  }
+  
+  
+  .market-section__subtitle {
+    margin: 62px 0 22px 0;
+  }
+
+  /* ---------------------------------------------------------- */
+  
+  .products__product-card {
+    margin: 0 0 0 0;
+  }
+  
+  .product-card {
+    display: flex;
+    flex-direction: column;
+    width: 208px;
+    padding: 0 9px 0 11px;
+  }
+
+  .product-card__picture {
+    margin: 0 0 33px 9px;
+  }
+
+  .product-card__picture img {
+    width: 172px;
+    height: 160px;
+  }
+}
+
+@media screen and (max-width: 950px) {
+
+  .products {
+    justify-content: space-around;
+}
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Expected indentation of 2 spaces (indentation)](https://app.codacy.com/gh/WebHeroSchool/Axion2/pullRequest?prid=5654147)